bundles/org.openhab.automation.jsscripting/pom.xml

```diff
@@ -67,7 +67,7 @@
               <goal>npm</goal>
             </goals>
             <configuration>
-              <arguments>install ${ohjs.version} webpack webpack-cli</arguments>
+              <arguments>install ${ohjs.version} webpack@5.75.0 webpack-cli@4.10.0</arguments> <!-- webpack-cli >= 5.0.0 doesn't properly process the given entrypoint -->
```

Comment:
FYI: This change sets the version of the build dependencies to the last versions that were working.
Before webpack-cli 5.0.0 was released, the build was working, since the release it is not. 4.10.0 is the last version before 5.0.0.

To avoid such build features in the future, I also manually set the webpack version to the latest (and working as well) version.